hatchet_sdk/loader.py

```diff
@@ -1,246 +1,147 @@
-
-        otel_exporter_oltp_protocol = get_config_value(
-            "otel_exporter_oltp_protocol", "HATCHET_CLIENT_OTEL_EXPORTER_OTLP_PROTOCOL"
-        )
-
-        worker_healthcheck_port = int(
-            get_config_value(
-                "worker_healthcheck_port", "HATCHET_CLIENT_WORKER_HEALTHCHECK_PORT"
-            )
-            or 8001
-        )
-
-        worker_healthcheck_enabled = (
-            str(
-                get_config_value(
-                    "worker_healthcheck_port",
-                    "HATCHET_CLIENT_WORKER_HEALTHCHECK_ENABLED",
-                )
-            )
-            == "True"
-        )
-
-        return ClientConfig(
-            tenant_id=tenant_id,
-            tls_config=tls_config,
-            token=token,
-            host_port=host_port,
-            server_url=server_url,
-            namespace=namespace,
-            listener_v2_timeout=listener_v2_timeout,
-            logger=defaults.logInterceptor,
-            grpc_max_recv_message_length=grpc_max_recv_message_length,
-            grpc_max_send_message_length=grpc_max_send_message_length,
-            otel_exporter_oltp_endpoint=otel_exporter_oltp_endpoint,
-            otel_service_name=otel_service_name,
-            otel_exporter_oltp_headers=otel_exporter_oltp_headers,
-            otel_exporter_oltp_protocol=otel_exporter_oltp_protocol,
-            worker_healthcheck_port=worker_healthcheck_port,
-            worker_healthcheck_enabled=worker_healthcheck_enabled,
-        )
-
-    def _load_tls_config(self, tls_data: Dict, host_port) -> ClientTLSConfig:
-        tls_strategy = (
-            tls_data["tlsStrategy"]
-            if "tlsStrategy" in tls_data
-            else self._get_env_var("HATCHET_CLIENT_TLS_STRATEGY")
-        )
-
-        if not tls_strategy:
-            tls_strategy = "tls"
-
-        cert_file = (
-            tls_data["tlsCertFile"]
-            if "tlsCertFile" in tls_data
-            else self._get_env_var("HATCHET_CLIENT_TLS_CERT_FILE")
-        )
-        key_file = (
-            tls_data["tlsKeyFile"]
-            if "tlsKeyFile" in tls_data
-            else self._get_env_var("HATCHET_CLIENT_TLS_KEY_FILE")
-        )
-        ca_file = (
-            tls_data["tlsRootCAFile"]
-            if "tlsRootCAFile" in tls_data
-            else self._get_env_var("HATCHET_CLIENT_TLS_ROOT_CA_FILE")
-        )
-
-        server_name = (
-            tls_data["tlsServerName"]
-            if "tlsServerName" in tls_data
-            else self._get_env_var("HATCHET_CLIENT_TLS_SERVER_NAME")
-        )
-
-        # if server_name is not set, use the host from the host_port
-        if not server_name:
-            server_name = host_port.split(":")[0]
-
-        return ClientTLSConfig(tls_strategy, cert_file, key_file, ca_file, server_name)
-
-    @staticmethod
-    def _get_env_var(env_var: str, default: Optional[str] = None) -> str:
-        return os.environ.get(env_var, default)
+            if not token:
```

Comment:
we may not need tenant id any more since the jwt wont be provisioned for a different tenant... something for a future pr